Automattic/newspack-newsletters
# chore: npm update

### All Submissions:






### Changes proposed in this Pull Request:



A [build on CI failed](https://app.circleci.com/pipelines/github/Automattic/newspack-newsletters/4733/workflows/c397b303-d9db-4032-aa6c-a9e4dbfe9db6/jobs/18452) with a webpack error (`HookWebpackError: error:0308010C:digital envelope routines::unsupported`). Running `npm update` fixes it. 
To resurrect this particular job, I've ssh'd into CI (reran job with SSH), ran `npm update`, and then `npm run build && npm run semantic-release`. It released the new version (v2.21.1) as intended.

I've also had to re-set prettier to the `npm:wp-prettier@2.6.2-beta-1` version and ran `npm run lint:js -- --fix`. 

### How to test the changes in this Pull Request:

1. Remove `node_modules` and `dist` folders
2. Run `npm i --legacy-peer-deps` and `npm run build`
3. Smoke test the plugin functionality (just to see if JS is built correctly)

### Other information:
